CMakeLists.txt

```diff
@@ -157,39 +157,30 @@ set(STORM_TEST_RESOURCES_DIR "${PROJECT_SOURCE_DIR}/resources/examples/testfiles
 # Auto-detect operating system.
 set(MACOSX 0)
 set(LINUX 0)
+set(APPLE_SILICON 0)
 if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
 	# Mac OS
 	set(OPERATING_SYSTEM "Mac OS")
         set(MACOSX 1)
+	if(${CMAKE_SYSTEM_PROCESSOR} MATCHES arm64)
+		set(APPLE_SILICON 1)
+	endif()
 elseif(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
-    # Linux
+	# Linux
 	set(OPERATING_SYSTEM "Linux")
         set(LINUX 1)
 elseif(WIN32)
-    # Assuming Windows.
-    set(OPERATING_SYSTEM "Windows")
+	# Assuming Windows.
+	set(OPERATING_SYSTEM "Windows")
 else()
-    message(WARNING "We are unsure about your operating system.")
-    set(OPERATING_SYSTEM "Linux")
-    set(LINUX 1)
+	message(WARNING "We are unsure about your operating system.")
+	set(OPERATING_SYSTEM "Linux")
+	set(LINUX 1)
 ENDIF()
 message(STATUS "Storm - Detected operating system ${OPERATING_SYSTEM}.")
 
 set(SHIPPED_CARL_USE_CLN_NUMBERS ON)
 set(SHIPPED_CARL_USE_GINAC ON)
-# Detect Apple Silicon and adjust settings
```

Comment:
I think we should still emit a warning, given that we still have various bugs on apple silicon....